### ISSUE TITLE
CERTIFICATE BY RECYCLER

### ISSUE THREAD

**Issue by 7597208625** [AppGrate/RecyclingSolutions-Issues#37]
1. There is no option where Recycler will have update about pending certificates. In certificate tab it can show list of shipments for which certificates are pending. Please check and think something (If we are going for this method admin should able to see pending shipment sorted with individual recycler).
2. Please introduce function of certificate approval by admin. Once Certificate approved by admin, Reccyler will not be able to edit the certificate issued.
3. Home -> Certificates -> "Select any shipment in Assigned Shipment - For Example Certificate 10" -> "Edit" Button -> Press
![WhatsApp Image 2020-12-30 at 6 30 34 PM](https://user-images.githubusercontent.com/73227626/103352780-2ede3e00-4acd-11eb-9713-6745526b8c86.jpeg)

4. Recycling Quantity should be auto populated based on selected shipments (Marked as red circle).
     
![WhatsApp Image 2020-12-30 at 6 32 18 PM](https://user-images.githubusercontent.com/73227626/103352891-7795f700-4acd-11eb-939d-6b62c2a3d296.jpeg)

5. Instead of Name it should be certificate number (Marked in red). 
![WhatsApp Image 2020-12-30 at 6 35 31 PM](https://user-images.githubusercontent.com/73227626/103353066-d3f91680-4acd-11eb-9430-47c3895c63cc.jpeg)

**Comment by prashantgithub82** [comment]
1. FIXED
2. We have removed recycler access for editing the certificate so explicit approval is not required.
3. FIXED
4. Working as expected. Whenever you add/remove shipment, quantity will be auto populated but if you can manually update that field.
5. FIXED

**Comment by 7597208625** [comment]
I am not able to understand certificate page. I have attached screen which a recycler can see.

![WhatsApp Image 2021-01-04 at 6 44 01 PM](https://user-images.githubusercontent.com/73227626/103538772-df2cb780-4ebc-11eb-8526-d5eb8001f8d3.jpeg)

1. Select any of the assigned certficate (Lets say Certficate 10) - In assigned certificate quanity is ZERO. How a certificate for zero quantity can be issued. Furhter, shipment or state or province is not there. So A certificate can not be issued without a state or province. Neither there is any shipment which is mentioned.
![WhatsApp Image 2021-01-04 at 6 46 58 PM](https://user-images.githubusercontent.com/73227626/103539010-45b1d580-4ebd-11eb-8c91-4f177cdcecbb.jpeg)

2. Draft tab is only showing "JB ECOTEX LLP". There is no other information. Further when we select any of the draft certificate, it is showing only invoice. No edit where state can be selected or number can be added nothing is there. 
![WhatsApp Image 2021-01-04 at 6 50 34 PM](https://user-images.githubusercontent.com/73227626/103539281-c7096800-4ebd-11eb-9fd5-2d6c087ddf61.jpeg)

So what is the meaning of draft here ?
Here, we should show information about shipments for which certificates are pending rather than draft. Format can be as below mentioned: 
PENDING CERTIFICATES
   1. Inv No. | State of Origin -   Quantity 
         Supplier | Recycler

3. Press "ADD" in certificate page -> Select state (Gujarat) -> Press "Select Shipment" ->  No shipment is shown.  However there are shipment received by JB ECOTEX from Gujarat in draft tab for which certificates are not isssued. 
![WhatsApp Image 2021-01-04 at 6 53 46 PM](https://user-images.githubusercontent.com/73227626/103539502-38491b00-4ebe-11eb-9fc6-f05a8a24f06f.jpeg)

4. Further quantity check which was added last time was good. If user select multiple shipment then quantity should be auto populated. 
5. Once Certificate are submitted by recycler then it can be marked as draft or pending approval by Admin. Once Certificate are approved are by admin then it will assigned and recycler can not change certificate. Untill admin does not approve the certificate recycler can withdraw the same anytime.
